servers/dsub/jobs/test/base_test_cases.py

```diff
@@ -18,8 +18,10 @@
 from jobs.models.job_metadata_response import JobMetadataResponse
 from jobs.models.query_jobs_response import QueryJobsResponse
 from jobs.models.query_jobs_request import QueryJobsRequest
+from jobs.models.aggregation_response import AggregationResponse
 
 DOCKER_IMAGE = 'ubuntu:14.04'
+_LABEL_MIN_COUNT_FOR_RANK = 3
```

Comment:
I would just set this to 0, I think the logic in this test is too complicated otherwise

Reply:
By using the filter label, we do not need to change this value when testing, reverting it to the last version.